#!/usr/bin/env python3
 
PACKAGE = "borealis_teaming_planner"
 
from dynamic_reconfigure.parameter_generator_catkin import *
 
gen = ParameterGenerator()
 

gen.add("agent_radius", double_t, 0, "Agent Radius",    .50, 0,   1)
gen.add("priority_penalty", double_t, 0, "Priority Penalty",    .03, -2,   2)
gen.add("triangle_size", double_t, 0, "Triangle Size",    3.0, 1,   5)
gen.add("line_size", double_t, 0, "Line Size",    1.5, 1,   5)
gen.add("point_removal_radius", double_t, 0, "Radius to remoove points ard agents",    0.9, 0.5,   3)

exit(gen.generate(PACKAGE, "borealis_teaming_planner", "borealis_teaming_planner_"))